extensions/wikia/CategoryPage3/templates/members-header.php

```diff
@@ -0,0 +1,15 @@
+<p class="category-page__total-number">
+	<?php /** @var $totalNumberOfMembers int */ ?>
+	<?= wfMessage( 'category-page3-total-number', $totalNumberOfMembers )->escaped() ?>
+</p>
+<ul class="category-page__alphabet-shortcuts">
+	<?php foreach ( $alphabetShortcuts as $shortcut ) : ?>
+		<li class="category-page__alphabet-shortcut<?= $shortcut['isActive'] ? ' is-active': '' ?>">
+			<?= Html::rawElement( 'a', [
```

Comment:
Can we use  `element` rather than `rawElement` to make sure it's escaped at the output?